client/templates/home.html

```diff
@@ -1,12 +1,32 @@
 <template name="home">
 
-  {{#if currentUser}}
-    {{> meetingSeriesList}}
+  {{#if sendVerificationEmailEnabled}}
+      {{#unless currentUser.emails.[0].verified}}
+          <p class="alert alert-warning">You need to verify your email address before.
```

Comment:
What if a not-verified user navigates (via meeting series id guessing or clicking on a link he received from someone else) to a different route?! Please make sure that this scenario blocks the user too.

Unfortunately in this template we have now a lot of code duplicates. Maybe we could get rid of them?